src/common/meta/src/key.rs

```diff
@@ -12,16 +12,102 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! This mod defines all the keys used in the metadata store (Metasrv).
+//! Specifically, there are these kinds of keys:
+//!
+//! 1. Table info key: `__table_info/{table_id}`
+//!     - The value is a [TableInfoValue] struct; it contains the whole table info (like column
+//!       schemas).
+//!     - This key is mainly used in constructing the table in Datanode and Frontend.
+//!
+//! 2. Table region key: `__table_region/{table_id}`
+//!     - The value is a [TableRegionValue] struct; it contains the region distribution of the
+//!       table in the Datanodes.
+//!
+//! All keys have related managers. The managers take care of the serialization and deserialization
+//! of keys and values, and the interaction with the underlying KV store backend.
+//!
+//! To simplify the managers used in struct fields and function parameters, we define a "unify"
+//! table metadata manager: [TableMetadataManager]. It contains all the managers defined above.
+//! It's recommended to just use this manager only.
+
+pub mod table_info;
+pub mod table_region;
 mod table_route;
 
+use std::sync::Arc;
+
+use snafu::ResultExt;
+use table_info::{TableInfoManager, TableInfoValue};
+use table_region::{TableRegionManager, TableRegionValue};
+
+use crate::error::{InvalidTableMetadataSnafu, Result, SerdeJsonSnafu};
 pub use crate::key::table_route::{TableRouteKey, TABLE_ROUTE_PREFIX};
+use crate::kv_backend::KvBackendRef;
 
 pub const REMOVED_PREFIX: &str = "__removed";
 
+const TABLE_INFO_KEY_PREFIX: &str = "__table_info";
+const TABLE_REGION_KEY_PREFIX: &str = "__table_region";
```

Comment:
Why not "__tr" "__ti"?　🤣

Reply:
Because it's the table id that follows these prefix, "__table_info/123" is more clear than "__ti/123".

Reply:
"__ti" Indeed is a bit unclear.